developer/openshift/gitops/argocd/pipeline-service-o11y/appstudio-prometheus/allow-argocd-to-manage.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: openshift-gitops-apply-prometheus
+rules:
+  - apiGroups:
+      - monitoring.rhobs
+    resources:
+      - monitoringstacks
+      - servicemonitors
+    verbs:
+      - get
+      - list
+      - patch
+      - create
+      - update
+      - delete
+  - apiGroups:
+    - monitoring.coreos.com
```

Comment:
I think you acknowledged this during scrum today, but I'll note it here just in case.

yamllint was unhappy here:

```
step-run-yamllint
yamllint 1.32.0
/workspace/source/developer/openshift/gitops/argocd/pipeline-service-o11y/appstudio-prometheus/allow-argocd-to-manage.yaml
  20:5      error    wrong indentation: expected 6 but found 4  (indentation)
  22:5      error    wrong indentation: expected 6 but found 4  (indentation)
```